PythonTool/GrabImagesFromSessions.py

```diff
@@ -0,0 +1,17 @@
+import os
```

Comment:
I don't know... :smile: That looks like an unfinished comment. Not important as this script is going away.